src/EnergyPlus/HybridEvapCoolingModel.cc

```diff
@@ -0,0 +1,1591 @@
+			}
+			else
+			{
+				ModeName = Alphas(inter_Alpha - 1);
+			}
+
+			curveID = -1;
+			if (lAlphaBlanks(inter_Alpha)) {
+				InitializeCurve(TEMP_CURVE, curveID);//as this is invalid curve id CalculateCurveVal will return a default 
+			}
+			else
+			{
+				curveID = GetCurveIndex(Alphas(inter_Alpha));
+				if (curveID == 0) {
+					ShowSevereError("Invalid " + cAlphaFields(inter_Alpha) + '=' + Alphas(inter_Alpha));
+					ShowContinueError("Entered in " + cCurrentModuleObject);
+					ErrorsFound = true;
+					InitializeCurve(TEMP_CURVE, -1);
+				}
+				else { InitializeCurve(TEMP_CURVE, curveID); }
+
+			}
+
+			inter_Alpha = inter_Alpha + 1;
+			//ZoneHybridUnitaryAirConditioner(UnitLoop).Tsa_curve_pointer.push_back(GetCurveIndex(Alphas(19)));
+
+			//A20, \field Mode0 Supply Air Humidity Ratio Lookup Table Name
+			curveID = -1;
+			if (lAlphaBlanks(inter_Alpha)) {
+				InitializeCurve(W_CURVE, curveID);//as this is invalid curve id CalculateCurveVal will return a default 
+			}
+			else
+			{
+				curveID = GetCurveIndex(Alphas(inter_Alpha));
+				if (curveID == 0) {
+					ShowSevereError("Invalid " + cAlphaFields(inter_Alpha) + '=' + Alphas(inter_Alpha));
+					ShowContinueError("Entered in " + cCurrentModuleObject);
+					ErrorsFound = true;
+					InitializeCurve(W_CURVE, -1);
+				}
+				else { InitializeCurve(W_CURVE, curveID); }
+
+			}
+			inter_Alpha = inter_Alpha + 1;
+			//A21, \field Mode0 System Electric Power Lookup Table Name
+			curveID = -1;
+			if (lAlphaBlanks(inter_Alpha)) {
+				InitializeCurve(POWER_CURVE, curveID);//as this is invalid curve id CalculateCurveVal will return a default 
+			}
+			else
+			{
+				curveID = GetCurveIndex(Alphas(inter_Alpha));
+				if (curveID == 0) {
+					ShowSevereError("Invalid " + cAlphaFields(inter_Alpha) + '=' + Alphas(inter_Alpha));
+					ShowContinueError("Entered in " + cCurrentModuleObject);
+					ErrorsFound = true;
+					InitializeCurve(POWER_CURVE, -1);
+				}
+				else { InitializeCurve(POWER_CURVE, curveID); }
+
+			}
+			//A21, \field Mode0 Supply Fan Electric Power Lookup Table Name
+			//A22, \field Mode0 External Static Pressure Lookup Table Name
+			//A23, \field Mode0 System Second Fuel Consumption Lookup Table Name
+			//A24, \field Mode0 System Third Fuel Consumption Lookup Table Name
+			//A25, \field Mode0 System Water Use Lookup Table Name
+			//Numbers(inter_Number) //N6 field Mode0 Outside Air Fraction
+								  //N7,field Mode0 Supply Air Mass Flow Rate Ratio
+			//injest OSAF and MsaRatio setting.
+
+			return false;
+		}
+		bool CMode::ParseMode(Array1D_string Alphas, Array1D_string cAlphaFields, Array1D< Real64 > Numbers, Array1D_string cNumericFields, Array1D<bool>  lAlphaBlanks, std::string cCurrentModuleObject)
+		{
+
+
+			if (!ValidateArrays(Alphas, cAlphaFields, Numbers, cNumericFields, cCurrentModuleObject))
+			{
+				ShowSevereError("Mode description array does not match mode number, in" + cCurrentModuleObject);
```

Comment:
These error messages need to be more descriptive.  You should have a specific object instance name at this point right?  That way you can point the user to a specific instance with the problem.  If the inputs for these objects are in the hundreds or thousands, it will be critical to help them figure out the exact problem.